requirements/REQUIREMENTS.md

```diff
@@ -0,0 +1,59 @@
+## User Roles
+
+### Unauthorized user
+
+Unauthorized users can sign in, view list of available songs and musical instruments. 
+
+### Authorized user
+
+Authorized user can sign out, rate added music, buy music sheets, buy premium account and get access to some musical instruments.
+
+### Premium user
+
+Premium user can do the same things as an authorized user and also have access to premium instruments and synth mapping technology.
+
+### Sheets Creator
+
+Sheets creator can do the same thing as an authorized user and add music sheets.
+
+### Moderator
+
+Moderator monitors the rights of musicians and protects our community from malicious content.
+
+## User Case Diagram
+
+<p align="center">
```

Comment:
What is the `Use case` near the sheets' creator?)

Reply:
```Use case``` is ```Add music sheets```. Fixed it.